src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -80,7 +82,9 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
             <strong>Note: </strong> Weights can be assigned for questions with existing questions too. To do that, edit the question that needs weights,
             if there are existing responses for that question, the additional statistics will automatically get calculated for the question.
           </p>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+            <tm-mcq-question-edit-details-form [isEditable]="true" [details]="exampleMCQWithWeightsQuestion"></tm-mcq-question-edit-details-form>
```

Comment:
> clicking on button or checking checkboxes do not change anything

Yup, it will not change because you do not subscribe to the event (Yup, we should not subscribe them here, you are doing it correct. I just mention here why it happens). 

Now I think you can consider disable the form.

Reply:
What do you mean by disabling the form? I don't see a way to submit this form currently.

Reply:
Setting `isEditable` to false will disable the form.

Reply:
I see. Will do the change.

Reply:
BTW, we have discussed with @madanalogy and conclude that we should use `tm-question-edit-form` instead. You can follow his recent PR for example.

Reply:
Ok no problem.